src/main/java/nl/knaw/dans/pf/language/ddm/api/Ddm2EmdCrosswalk.java

```diff
@@ -87,9 +87,27 @@ public EasyMetadata createFromValidated(final File file) throws CrosswalkExcepti
      * @throws CrosswalkException
      */
     public EasyMetadata createFrom(final String xml) throws CrosswalkException {
-        return validateEMD(walk(ddmValidator, xml, newTarget()));
+        return validateEMD(walk(ddmValidator, replaceSurrogatePairsWithTempCode(xml), newTarget()));
     }
 
+    private String replaceSurrogatePairsWithTempCode(String xmlStr) {
```

Comment:
please add documentation to this method. Otherwise it won't be clear in the future why we did this. I found the explanation you gave in the email, earlier this week, very helpful and enlightening. Maybe you could add a similar explanation here as documentation.